hunter_base/include/hunter_base/hunter_messenger.hpp

```diff
@@ -262,31 +263,40 @@ class HunterMessenger {
 
   void controlLoop(double& len_vel, double& anu_vel) {
 
-    // auto current_time = this->now();
-    double dt = (current_time_ - last_time_).seconds();
-    // last_time_ = current_time;
+    auto current_time = node_->get_clock()->now();
+    double dt = 1.0/50.0;
```

Comment:
Why the dt is fixed at 1/50? and why are you no longer using the last_time_control_law_ variable to calculate the dt?

Reply:
`dt` can not be fixed like this, as this is the discrete-time used to calculate the control output, usually the control output timer is performed on timer interrupt to ensure accurate timing, as if this is missed up, the controller behaviour is not predictable